src/pages/events.js

```diff
@@ -11,6 +12,11 @@ export default class Events extends Component {
     return (
       <div>
         <h1>Hi from the events page</h1>
+        <ImageBanner
+          imageSrc="https://static.guim.co.uk/sys-images/Guardian/Pix/pictures/2014/4/11/1397210130748/Spring-Lamb.-Image-shot-2-011.jpg"
```

Comment:
Is this a placeholder? I guess we should ensure images are either bundled as assets in our app or hosted on Contentful